storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -215,13 +215,13 @@ Options
 .. option:: --incremental-lsn=LSN
 
    When creating an incremental backup, you can specify the log sequence number
-   (:term:`LSN`) instead of specifying :option:`--incremental-basedir`. For
-   databases created by *MySQL* and *Percona Server* 5.0-series versions,
-   specify the :term:`LSN` as two 32-bit integers in high:low format. For
-   databases created in 5.1 and later, specify the :term:`LSN` as a single
-   64-bit integer.  **ATTENTION**: If a wrong LSN value is specified (a user
-   error which |Percona XtraBackup| is unable to detect), the backup will be
-   unusable. Be careful!
+   (:term:`LSN`) instead of specifying
+   :option:`xtrabackup --incremental-basedir`. For databases created by *MySQL*
+   and *Percona Server* 5.0-series versions, specify the :term:`LSN` as two
+   32-bit integers in high:low format. For databases created in 5.1 and later,
```

Comment:
hi:lo format was good for old times when xtrabackup_50 was in use. Now when xtrabackup is single binary and 5.0 isn't supported this is no longer true. LSN is just a 64bit int.